### PR TITLE
Upgrading to Node version 16

### PR DESCRIPTION
**Pull request by rakeshkumar1019**
Upgrading to Node version 16

### CODE REVIEW

**Review by GuruprasadAnk1996 — DISMISSED** [review]

**Review by swarnadipa-dev — DISMISSED** [review]

File: packages/nuxeo-web-ui-ftest/pages/login.js [hunk 1]
```diff
@@ -9,11 +9,27 @@ export default class Login {
     $('#password').setValue(password);
   }
 
-  submit() {
+  submitSync() {
     $('[name="Submit"]').waitForDisplayed();
     return $('[name="Submit"]').click();
   }
 
+  async setusername(username) {
```
Comment (satishyadav-dev) [inline packages/nuxeo-web-ui-ftest/pages/login.js]: variable  and method name should be in camel case e.g. "setUserName" and "userName". 
correct it for other places as well.

Comment (rakeshkumar1019) [reply packages/nuxeo-web-ui-ftest/pages/login.js]: Thank you @satishyadav-dev

**Review by GuruprasadAnk1996 — DISMISSED** [review]

**Review by swarnadipa-dev — DISMISSED** [review]

**Review by satishyadav-dev — DISMISSED** [review]

**Review by Pranit-Sotre — DISMISSED** [review]

File: packages/nuxeo-web-ui-ftest/pages/login.js [hunk 2]
```diff
@@ -9,11 +9,27 @@ export default class Login {
...
+
+  async setPassword(password) {
+    const passwd = $('#password');
+    await (await passwd).setValue(password);
+  }
+
+  async submit() {
+    const submitBtn = $("[name='Submit']");
+    await (await submitBtn).click();
+    submitBtn.click();
```
Comment (rajatgoel-gl) [inline packages/nuxeo-web-ui-ftest/pages/login.js]: remove multiple times calling of click event.

File: package.json [hunk 3]
```diff
@@ -74,7 +74,7 @@
     "prebuild": "npm-run-all prepare:*",
     "prepare:tmp": "rm -rf .tmp",
     "prepare:i18n": "node scripts/merge-messages.js",
-    "prepare:workbox": "workbox copyLibraries .tmp && mv .tmp/workbox-v$npm_package_devDependencies_workbox_cli .tmp/workbox",
+    "prepare:workbox": "workbox copyLibraries .tmp && mv .tmp/workbox-v3.6.3 .tmp/workbox",
```
Comment (rajatgoel-gl) [inline package.json]: Do we really need to hardcode it?

File: plugin/web-ui/addon/build.xml [hunk 4]
```diff
@@ -33,8 +33,6 @@ limitations under the License.
   <target name="install" description="Install npm depedencies" unless="skipInstall">
     <exec executable="${cmd.npm}" failonerror="true">
       <arg value="install" />
-      <!-- remove next line once NXBT-3340 is closed -->
-      <arg value="--@nuxeo:registry=&quot;https://packages.nuxeo.com/repository/npm-public&quot;" />
```
Comment (rajatgoel-gl) [inline plugin/web-ui/addon/build.xml:37]: Are we sure we dont need it?

**Review by rajatgoel-gl — DISMISSED** [review]

File: packages/nuxeo-web-ui-ftest/pages/login.js (hunk 2 above)
Comment (rakeshkumar1019) [reply packages/nuxeo-web-ui-ftest/pages/login.js]: Done, thank you for comment

File: package.json (hunk 3 above)
Comment (rakeshkumar1019) [reply package.json]: here with `v$npm_package_devDependencies_workbox_cli` we are getting error that `workbox-v3.6.3` No such file or directory.

File: plugin/web-ui/addon/build.xml (hunk 4 above)
Comment (rakeshkumar1019) [reply plugin/web-ui/addon/build.xml:37]: yes, its working without this.

**Review by GuruprasadAnk1996 — DISMISSED** [review]

**Review by Nishant0928 — DISMISSED** [review]

**Review by Pranit-Sotre — DISMISSED** [review]

**Review by swarnadipa-dev — DISMISSED** [review]

**Review by Nisha-QA — DISMISSED** [review]

**Review by satishyadav-dev — DISMISSED** [review]

**Review by poonamyadav252 — DISMISSED** [review]
Please close PR after the all required approvals

**Pull request closed without merging**